graphql-compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/PluginConstants.java

```diff
@@ -61,7 +62,9 @@
                 "http:Listener and graphql:ListenerConfiguration are mutually exclusive", "GRAPHQL_110",
                 DiagnosticSeverity.ERROR),
         INVALID_RETURN_TYPE_MULTIPLE_SERVICES("Invalid return type with multiple indistinct services",
-                "GRAPHQL_111", DiagnosticSeverity.ERROR);
+                "GRAPHQL_111", DiagnosticSeverity.ERROR),
+        INVALID_RESOURCE_PATH("Invalid field name. GraphQL field Name must not begin with \"__\", " +
```

Comment:
This name is misleading. Shall we rename this to something like `INVALID_FIELD_NAME`? 
```suggestion
        INVALID_FIELD_NAME("Invalid field name. A GraphQL field Name must not begin with \"__\", " +
```